src/amo/components/LandingPage.scss

```diff
@@ -3,3 +3,69 @@
 .LandingPage {
   padding: $padding-page;
 }
+
+.Addon-Header {
+  margin: 10px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.Icon-persona {
+  background: url('~amo/img/icons/themes.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 50px;
+  margin-bottom: 15px;
+  width: 41.5px;
+}
+
+.Icon-extension {
+  background: url('~amo/img/icons/extensions.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 47.8px;
+  margin-bottom: 15px;
+  width: 49px;
+}
+
+.Addon-Header-Text {
+  display: inline-block;
+  margin-left: 20px;
+  text-align: left;
+  width: 250px;
+}
+
+.Addon-Header-Text-Bold {
+  font-size: 20px;
+  font-weight: bold;
+  padding: 0;
+}
+
+.Addon-Header-Text-Content {
+  margin-top: 5px;
+}
+
+.Browse-Button {
+  background-color: white;
+  border-radius: 9px;
+  color: blue;
+  height: 55px;
+  font-size: 12px;
+  margin-left: calc(50% - 150px);
+  margin-bottom: 2em;
+  text-transform: uppercase;
+  width: 300px;
+}
+
+.Icon-browse {
+  background: url('~amo/img/icons/bullet-list.svg') 50% 50% no-repeat;
+  background-size: cover;
+  height: 30px;
+  margin-right: 1em;
+  width: 30px;
+}
+
+.Browse-by-Category-Text {
+  bottom: 10px;
+  position: relative;
```

Comment:
I tired it with margin, padding and line-height but that didn't work. Then I tried the position which seemed to work somehow.
I'll have a look at the home page button styling and try to replicate that here too.